### DISCUSSION TITLE
Display support request:  GC9107

### DISCUSSION THREAD

**Discussion by nikthefix** [Bodmer/TFT_eSPI#1922] [Compatible displays and setup files]
Display support request:  GC9107 
https://www.tindie.com/products/lilygo/lilygo-t-085-inch-lcd-module/

I've ordered one and will try with the the GC9A01 driver to determine mods if any.

**Comment by Bodmer** [comment]
The init sequence here looks quite different so the GC9A01 driver probably will not work and a new driver will need to be created:
https://github.com/Xinyuan-LilyGO/T-QT/blob/main/example/lcd/lcd_driver.cpp

**Comment by johnatannvmd** [comment]
According to the T-QT [examples](https://github.com/Xinyuan-LilyGO/T-QT/blob/main/example/lcd2/lcd2.ino), you could try to use st7789 driver, because:
> //gc9107 is basically the same as st7789 driver initialization

**Comment by tyeth** [comment]
Just to say the situation is changed and now it is using the GC9A01:
https://github.com/Xinyuan-LilyGO/T-QT/blob/f25f75f2cead0a573af62a2998ad9bedc74ee8dd/lib/TFT_eSPI/User_Setup_Select.h#L140
https://github.com/Xinyuan-LilyGO/T-QT/blob/f25f75f2cead0a573af62a2998ad9bedc74ee8dd/lib/TFT_eSPI/User_Setups/Setup211_LilyGo_T_QT_Pro_S3.h#L4
